### PR TITLE
Updated to look for components in the western-up-react node module

### PR DESCRIPTION
**Pull request by ModulesUnraveled**

### CODE REVIEW

File: .storybook/config.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ addDecorator(withA11y);
 import '../src/components/styles.scss';
 
 // automatically import all files ending in *.stories.js
-configure(require.context('../src/components', true, /\.stories\.js$/), module);
+configure(require.context('western-up-react', true, /\.stories\.js$/), module);
```
Comment (nJim) [inline .storybook/config.js:19]: @ModulesUnraveled I had to change this from `western-up-react ` to `../western-up-react` for storybook to discover the western up stories. Not sure if anyone else will hit this roadblock or if I set up the site incorrectly.

File: .gitignore [hunk 2]
```diff
@@ -65,5 +65,9 @@ public
 yarn-error.log
 .pnp/
 .pnp.js
+
 # Yarn Integrity file
 .yarn-integrity
+
+# Yarn Workspace Directories
+western-up-react
```
Comment (evanmwillhite) [inline .gitignore:73]: Is this needed? I didn't think we were using workspaces here anymore.

File: .storybook/config.js (hunk 1 above)
Comment (evanmwillhite) [reply .storybook/config.js:19]: Assuming we're not using workspaces, we should just be able to use `western-up-react`

**Review by evanmwillhite — APPROVED** [review]
See my comments, but otherwise all looks great

File: .storybook/config.js (hunk 1 above)
Comment (ModulesUnraveled) [reply .storybook/config.js:19]: I've been using workspaces and it works as is. I know what the problem was for you Jim.

File: .gitignore (hunk 2 above)
Comment (ModulesUnraveled) [reply .gitignore:73]: I've been using workspaces for the initial development of the components, that's why this is here

**Comment by evanmwillhite** [comment]
I think workspaces are being used initially here, so I'm gonna merge

**Pull request merged**